hashtagmap-batch/src/main/java/com/songpapeople/hashtagmap/job/InstagramBatchWriter.java

```diff
@@ -0,0 +1,55 @@
+package com.songpapeople.hashtagmap.job;
+
+import com.songpapeople.hashtagmap.instagram.domain.model.Instagram;
+import com.songpapeople.hashtagmap.instagram.domain.model.InstagramPost;
+import com.songpapeople.hashtagmap.instagram.domain.repository.InstagramRepository;
+import com.songpapeople.hashtagmap.instagram.domain.repository.instagramPost.InstagramPostRepository;
+import com.songpapeople.hashtagmap.place.domain.model.Place;
+import com.songpapeople.hashtagmap.scheduler.CrawlingResult;
+import lombok.RequiredArgsConstructor;
+import org.springframework.batch.item.ItemWriter;
+import org.springframework.stereotype.Component;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+@RequiredArgsConstructor
+@Component
+public class InstagramBatchWriter implements ItemWriter<Optional<CrawlingResult>> {
+    private final InstagramRepository instagramRepository;
+    private final InstagramPostRepository instagramPostRepository;
+
+    @Override
+    public void write(List<? extends Optional<CrawlingResult>> items) {
+        List<CrawlingResult> crawlingResults = items.stream()
+                .filter(Optional::isPresent)
+                .map(item -> item.orElseThrow(NullPointerException::new))
+                .collect(Collectors.toList());
+        saveCrawlingResult(crawlingResults);
+    }
+
+    @Transactional
+    void saveCrawlingResult(List<CrawlingResult> crawlingResults) {
```

Comment:
`@Transcational` 어노테이션은 public 메서드에서 사용해야합니다
```suggestion
    public void saveCrawlingResult(List<CrawlingResult> crawlingResults) {
```